python/ray/tune/tests/test_sample.py

```diff
@@ -240,8 +240,10 @@ def testConvertBayesOpt(self):
         bayesopt_config = {"b/z": (1e-4, 1e-2)}
         converted_config = BayesOptSearch.convert_search_space(config)
 
-        searcher1 = BayesOptSearch(space=converted_config, metric="none")
-        searcher2 = BayesOptSearch(space=bayesopt_config, metric="none")
+        searcher1 = BayesOptSearch(
+            space=converted_config, metric="none", mode="max")
+        searcher2 = BayesOptSearch(
+            space=bayesopt_config, metric="none", mode="max")
```

Comment:
tests look fine, though in the future, we will want to explicitly test that the expected error is raised :)